tests/routes/role.js

```diff
@@ -4,6 +4,8 @@ import chaiHttp  from 'chai-http'
 import { Role } from "../../models";
 import server from "../../index";
 
+process.env.NODE_ENV = 'test';
```

Comment:
How about we modify our npm test script to set the variable before running tests, so we don't have to  include this line in every file?
i.e. 
 ...
  "scripts": {
    "test": "NODE_ENV=test nyc mocha tests/**/*.js"
  }
  ...

Reply:
Correct. I will move that